torch/csrc/api/src/nn/options/normalization.cpp

```diff
@@ -1 +1,9 @@
 #include <torch/nn/options/normalization.h>
+
+namespace torch {
+namespace nn {
+
+LayerNormOptions::LayerNormOptions(std::vector<int64_t> normalized_shape) : normalized_shape_(normalized_shape) {}
```

Comment:
clang-tidy complains:
```
{
    path: 'torch/csrc/api/src/nn/options/normalization.cpp',
    start_line: 6,
    end_line: 6,
    start_column: 36,
    end_column: 36,
    annotation_level: 'failure',
    message: '[[modernize-pass-by-value]] warning: pass by value and use std::move'
  }
```
for this one I think we can do `normalized_shape_(std::move(normalized_shape))`.